fastboot-server.js

```diff
@@ -36,7 +39,7 @@ cluster(function() {
   if (assetPath) {
     app.get('/', fastboot);
     app.use(staticGzip(assetPath)),
-    app.use(express.static(assetPath));
+      app.use(express.static(assetPath));
```

Comment:
accident?

Reply:
@kellyselden prettier re-formatted because the previous line is a comma 😱